Pocket/pocket-ios
# Home - Recommendations

## Goal
* This PR completes the Recommendations section of Home, including
    - Save/unsave action with real time changes
    - iPad UI

> [!NOTE]
> Navigation and related (title, see all buttons etc) will be added in future PRs
> Recent saves and Shared With You sections will be added in future PRs

## Test Steps
* Build/run
* Go to SwiftUI Home
* Make sure the Recommendations section is consistent with the existing Home
* Test on iPad full and split screen as well
